### PR TITLE
chore: update CRI client runtime version to v1 (test with multiple k8s versions)

### PR DESCRIPTION
**Pull request by pmengelbert**

### CODE REVIEW

**Comment by sozercan** [comment]
@cpuguy83 suggested, "If we make it so we can fallback to v1alpha2 in case v1 is not available that would work."

File: api/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -28,6 +28,16 @@ import (
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *Image) DeepCopyInto(out *Image) {
 	*out = *in
+	if in.Names != nil {
```
Comment (sozercan) [inline api/v1alpha1/zz_generated.deepcopy.go:31]: why did this get updated?

Comment (pmengelbert) [reply api/v1alpha1/zz_generated.deepcopy.go:31]: `make generate` hasn't been run in a while, and it got run as part of `make test`. The changes in #477 caused this, as it changed the definition of the `Image` type. We should probably implement a codegen check for the CI, similar to the manifest check we already have

Comment (pmengelbert) [reply api/v1alpha1/zz_generated.deepcopy.go:31]: Just added that check to the CI for this PR

Comment (sozercan) [reply api/v1alpha1/zz_generated.deepcopy.go:31]: that's what i was thinking. awesome, thank you!

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -40,9 +40,11 @@ const (
 	EnvEraserContainerRuntime = "ERASER_CONTAINER_RUNTIME"
 )
 
-type ExclusionList struct {
-	Excluded []string `json:"excluded"`
-}
+type (
```
Comment (cpuguy83) [inline pkg/utils/utils.go]: Why this change? This is pretty abnormal for go.

Comment (pmengelbert) [reply pkg/utils/utils.go]: Slipped through (there was other stuff in there before and I moved it out).

**Review by cpuguy83 — APPROVED** [review]
